include/seccomp.h.in

```diff
@@ -196,6 +196,18 @@ struct scmp_arg_cmp {
 #define SCMP_ARCH_PARISC	AUDIT_ARCH_PARISC
 #define SCMP_ARCH_PARISC64	AUDIT_ARCH_PARISC64
 
+/**
+ * The RISC-V architecture tokens
+ */
+#define SCMP_ARCH_RISCV64	AUDIT_ARCH_RISCV64
+/* RISC-V support for audit was merged in 5.0-rc1 */
+#ifndef AUDIT_ARCH_RISCV64
+#ifndef EM_RISCV
+#define EM_RISCV		243
+#endif /* EM_RISCV */
+#define AUDIT_ARCH_RISCV64	(EM_RISCV|__AUDIT_ARCH_64BIT|__AUDIT_ARCH_LE)
```

Comment:
In keeping with the other tokens in seccomp.h, I would have expected `AUDIT_ARCH_RISCV64` to be defined before it was used.  However, if that is the only nit in this review we can move that while merging.